<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1046</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>操作</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>输出路径：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtOutputDir"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="btnFolder">
            <property name="text">
             <string>选择文件夹</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnFile">
            <property name="text">
             <string>选择文件</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnOutputDir">
            <property name="text">
             <string>输出路径</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="labelWidth">
            <property name="text">
             <string>宽</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditWidth">
            <property name="text">
             <string>1280</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelHeight">
            <property name="text">
             <string>高</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditHeight">
            <property name="text">
             <string>720</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>码率</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditBiterate">
            <property name="text">
             <string>1M</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="toolTip">
             <string>1-58，越小，压缩质量越好，但是内存越大</string>
            </property>
            <property name="text">
             <string>品质</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtCrf"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="toolTip">
               <string>1-58，越小，压缩质量越好，但是内存越大</string>
              </property>
              <property name="text">
               <string>速度</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="btnStart">
          <property name="text">
           <string>开始转换</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="txtShow"/>
        </item>
       </layout>
      </widget>
      <widget class="QListWidget" name="listWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1046</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSet">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="actionOptions"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSet"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionDelete">
   <property name="text">
    <string>删除</string>
   </property>
   <property name="toolTip">
    <string>删除该项</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>清空</string>
   </property>
   <property name="toolTip">
    <string>清空所有选项</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>选项</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
